src/components/Choice/src/ChoiceCard.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+	<m-card
+		:class="[
+			$s.ChoiceCard,
+			{
+				[$s.selected]: isSelected,
+				[$s.disabled]: isDisabled,
+			},
+		]"
+		:disabled="isDisabled"
+		@click="selectSelf"
+	>
+		<slot />
```

Comment:
Could we expose whether or not a card is selected to the slot? We use this pattern in checkout and it would make it easier to show/collapse the card content based on selection.